web_src/js/components/RepoActionView.vue

```diff
@@ -193,6 +197,11 @@ const sfc = {
       await this.fetchPost(`${jobLink}/rerun`);
       window.location.href = jobLink;
     },
+    // rerun workflow
+    rerun() {
+      this.fetchPost(`${this.run.link}/jobs/-1/rerun`);
```

Comment:
Is `0` already used? If no, I guess `0` seems better. If yes, I'd like to use `/job/current/rerun`.

The -1 is used for "undefined" in many places.

Reply:
IIRC, `0` is alredy used. Why not `${this.run.link}/rerun`?

Reply:
@wolfogre 
if use this link, jobindex will be 0, it is same to the event of ‘rerun the first job of this workflow’

@wxiaoguang 
It is defined as job index, 0 means the first job of this workflow. 
There are two inputs, run index and job index, i don’t know why use index instead of id. 
if job index is -1, it means this input is undefined, we will get only one valid input.
Then gitea will rerun all jobs.
maybe use id instead of index is the best way to fix this problem.

Reply:
You may misunderstood, I mean `${this.run.link}/rerun`, not `${jobLink}/rerun`

Reply:
I see. 
`runIndex` is as same as `Id`, but it is not global `ActionRun.Id`, it is an `runId` in the repo.
But `jobIndex` is actually a job list index:
https://github.com/go-gitea/gitea/blob/cf465b472166ccf6d3e001e3043e4bf43e16e6b3/routers/web/repo/actions/view.go#L379-L382

Reply:
Convert into `${this.run.link}/rerun` now. 😄

Reply:
Is this resolved?